how-to-guides/data-versioning/notebooks/Compare_model_training_runs_on_dataset_versions.ipynb

```diff
@@ -112,7 +137,7 @@
     "TRAIN_DATASET_PATH = \"../datasets/tables/train.csv\"\n",
     "TEST_DATASET_PATH = \"../datasets/tables/test.csv\"\n",
     "\n",
-    "PARAMS = {\n",
+    "params = {\n",
```

Comment:
Curious why this change? I thought parameters would usually be treated as constants

Reply:
these are the ones we are logging and can change between runs